addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -8,14 +8,14 @@
                 <t t-call="{{ constructor.popoverTemplate }}"/>
             </t>
             <t t-if="props.type === 'key'">
-                <div t-att-data-template="depTemplate" data-html="true" title="Dependencies" t-ref="action">
+                <div t-att-data-bs-template="depTemplate" data-bs-html="true" title="Dependencies" t-ref="action">
                     It looks like your file is being called by
                     <a href="#" t-on-click="showDependencies"><t t-esc="depText" /></a>.
                     Changing its name will break these calls.
                 </div>
             </t>
             <t t-else="">
-                <div t-att-data-template="depTemplate" data-html="true" title="Dependencies" t-ref="action">
+                <div t-att-data-bs-template="depTemplate" data-bs-html="true" title="Dependencies" t-ref="action">
```

Comment:
Should this be a dedicated commit? I guess it is a BS5 fix, not an OWL conversion?

Reply:
Added in a separate commit :+1: